client/src/components/PetProfile.jsx

```diff
@@ -0,0 +1,252 @@
+// todo: search material tailwind documentation to find a way to set a default tab
+// todo: change styling for tooltip info
+// todo: change color scheme
+// todo: add title font
+import {
+  Card,
+  CardHeader,
+  Typography,
+  Tooltip,
+  IconButton,
+  CardBody,
+  Tabs,
+  TabsHeader,
+  Tab,
+  TabPanel,
+  TabsBody,
+  CardFooter,
+  Button,
+} from "@material-tailwind/react";
+
+export default function PetProfile({
+  petname,
+  species,
+  breed,
+  sex,
+  altered,
+  birthday,
+  weight,
+  physicaldesc,
+  checkedin,
+  staytype,
+  owner,
+}) {
+  // get string to indicate altered/unaltered
+  function getAltered(altered) {
```

Comment:
I'd recommend you pull out these little utilities into their own file (or files). That way they can be easily unit tested, re-used, and keeps the component focused on just rendering HTML.

Reply:
I'm already seeing how this would be useful as I'm adding more components--I plan on pulling these out later today :)